andela/mag-ah-frontend
# #161338201 Harmonise UI and clear input fields on submit

#### What does this PR do?
Harmonizes signup, login, reset password and enter new password modals

#### Description of tasks to be completed
- Harmonize all authentication modals
- Clear input fields on submit

#### How should this be manually tested?
- `Git clone https://github.com/andela/mag-ah-frontend`
- `git checkout bg-clear-state-on-submit-161338201`
- `npm install` to install all the dependancies
- Click on get started
- Click on login
- Click on forgot password
- The UIs should look similar

#### What are the relevant pivotal tracker stories?
[Fix CORS whitelist](https://www.pivotaltracker.com/story/show/161338201)

#### Screenshots
Sign up Form

<img width="658" alt="screen shot 2018-10-22 at 16 53 24" src="https://user-images.githubusercontent.com/12625365/47296723-d2915100-d61b-11e8-9ea6-e9f7ffcae1eb.png">


Sign in Form

<img width="667" alt="screen shot 2018-10-22 at 16 53 36" src="https://user-images.githubusercontent.com/12625365/47296693-bc839080-d61b-11e8-9ce8-925693cb88af.png">

Forgot Password Form

<img width="577" alt="screen shot 2018-10-22 at 16 54 34" src="https://user-images.githubusercontent.com/12625365/47296627-89410180-d61b-11e8-82f0-036a2f7d17e9.png">

Update password form 

<img width="585" alt="screen shot 2018-10-22 at 16 55 43" src="https://user-images.githubusercontent.com/12625365/47296585-631b6180-d61b-11e8-94b9-d298c491dc9d.png">
